install/charts/templates/configure-karydia-webhook.sh.tpl

```diff
@@ -35,6 +35,7 @@ metadata:
 webhooks:
   - name: {{ .Values.metadata.apiGroup }}
     failurePolicy: Ignore
+    timeoutSeconds: 10
```

Comment:
This field is supported only for k8s `>=v1.14.x` - If you are planning to support older versions, please set this field only on clusters `>=v1.14.x`.
If you switch from the `v1beta1` to the `v1` API, the default is 10 seconds.

Reply:
You've got a point here. I hoped older k8s versions would just ignore the timeout then, but they completely fail to register the webhooks with `timeoutSeconds` configured. Since we do not need hooks for `node` resources yet, we decided to remove them from registration and keep the default timeout.

Reply:
@dacappo the problem was not just nodes it was also with pods. as calico pods need to start before karydia can be scheduled. Therefore you have to also remove the pods which are essential for karydia.
SO I guess the only way to solve both issues is by setting the timeout and somehow get the current k8s version

Reply:
@schrodit I see. Then we are running into problems for clusters `<v1.14.x` anyway. We will check for the Kubernetes version in helm and set the `timoutSeconds` for newer ones. Since (at least for my test clusters) all pods came up when removing `nodes`, we will go with that.

Reply:
Following code should deploy the expected behavior:
```
    {{- if semverCompare ">= 1.14" .Capabilities.KubeVersion.GitVersion }}
    timeoutSeconds: 10
    {{- end}}
```

I tried it with a server running `1.13.12` with a `timeout` not set  and a server running `1.16.2` with a `timeout` set to `10` (as expected).

However, while testing this feature I encountered version problems with the `objectSelector` that is also not supported under version `1.14` or some other version. This feature would also need a version check, e.g.
```
    {{- if and (semverCompare ">= 1.14" .Capabilities.KubeVersion.GitVersion) (.Values.exclusionObjectLabels) }}
    objectSelector:
      matchExpressions:
        {{- range .Values.exclusionObjectLabels }}
        - key: {{ .key | quote }}
        {{- if .values }}
          operator: NotIn
          values:
          {{- range .values }}
          - {{ . | quote }}
          {{- end }}
        {{- else }}
          operator: DoesNotExist
        {{- end }}
        {{- end }}
    {{- end }}
```